modules/services/redshift.nix

```diff
@@ -87,6 +87,15 @@ in
       '';
     };
 
+    trayApplet = mkOption {
```

Comment:
I think this option could be simply called `tray` to match the udiskie and possibly syncthing (#188) modules.

Reply:
Makes sense. I've amended the PR :)